layershellev/src/lib.rs

```diff
@@ -1387,6 +1388,18 @@ impl<T> Dispatch<xdg_surface::XdgSurface, ()> for WindowState<T> {
         }
     }
 }
+
+impl<T> Dispatch<WlRegion, ()> for WindowState<T> {
```

Comment:
I think it can use dispatch_noop! macro here

Reply:
Thanks, I didn't understand how that worked, it seems better